packages/studio-base/src/panels/VariableSlider/index.tsx

```diff
@@ -35,11 +35,19 @@ function VariableSliderPanel(props: Props): JSX.Element {
   const { min = 0, max = 10, step = 1 } = sliderProps;
   const globalVariableValue = globalVariables[globalVariableName];
   const theme = useTheme();
+  const [sliderValue, setSliderValue] = React.useState<number | number[]>(
```

Comment:
nit: we tend to use the the react builtin hooks without the "React." prefix.
```suggestion
  const [sliderValue, setSliderValue] = useState<number | number[]>(
```